app/assets/stylesheets/layout/_toolbar.sass

```diff
@@ -180,7 +180,8 @@ $nm-color-success-background: #d8fdd1
 
 .title-container
   flex: 1
-  padding: 0
+  overflow: hidden
```

Comment:
This leads to something like this:
<img width="424" alt="bildschirmfoto 2018-10-24 um 10 06 29" src="https://user-images.githubusercontent.com/7457313/47415746-7ee64b00-d774-11e8-8735-de798a578da8.png">